debian/README.Debian

```diff
@@ -0,0 +1,18 @@
+NOTE: The Debian version of libpam-radius-auth uses as default configuration
```

Comment:
We don't need debian-specific notes in our packages.

Just change the default configuration file to `/etc/pam_radius_auth.conf`.